WalletWasabi/Helpers/ImportWalletHelper.cs

```diff
@@ -52,9 +52,12 @@ private static KeyManager GetKeyManagerByWasabiJson(WalletManager manager, strin
 
 	private static KeyManager GetKeyManagerByColdcardJson(WalletManager manager, JObject jsonWallet, string walletFullPath)
 	{
-		var xpubString = jsonWallet["ExtPubKey"]?.ToString()
+		var segwitXpubString = jsonWallet["ExtPubKey"]?.ToString()
 			?? throw new ArgumentNullException($"Can't get KeyManager, ExtPubKey was null.");
 
+		var taprootXpubString = jsonWallet["TaprootExtPubKey"]?.ToString()
+		                       ?? throw new ArgumentNullException($"Can't get KeyManager, TaprootExtPubKey was null.");
```

Comment:
No, this is a huge bug introduced by me!

Reply:
Okay. Fixed. The idea is to support importing wallets created by ColdCard for Wasabi, even when currently they create the skeleton for segwit only (because it was the only one that we supported). If we start supporting taproot then they will be able to include the `TaprootExtPubKey` field in their skeleton wallet too.

Anyway, we can remove this and include it in the future if we need it.